hack/testing/rsyslog/parse_json.rulebase

```diff
@@ -1 +1,2 @@
-rule=:%payload:json%
+version=2
+rule=:%.:json%
```

Comment:
Wow.  This is very simple and clean.
I have one question.  If a log like this is given:
`{"log":"{\"message\":\"some_message\"}", "others":"{\"foo\":\"bar\"}", "top":"level"}`
the parsed result would be
`{"message":"some_message", "foo":"bar", "top":"level"}`
?  Or are there any way to specify just process the value of "log"?  (I guess there's no such use case for now, though... :)

Reply:
This makes it parse only the value of `$!log` - it was already in there so it didn't have to change, only the rulebase file:
```
          action(type="mmnormalize" ruleBase="/etc/rsyslog.d/viaq/parse_json.rulebase" variable="$!log") 
```
so the parsed result is
```
{"message":"some_message", "others":"{\"foo\":\"bar\"}", "top":"level","log":"{\"message\":\"some_message\"}"}
```
I still need to add in the code to delete `$!log` or `$!MESSAGE` if it was parsed correctly.

Reply:
or keep it with naming the key to "original_log":"{\"message\":\"some_message\"}?

Reply:
or yes, keep it
```
set $!pipeline_metadata.collector.original_raw_message = $!log;
unset $!log;
```
or something like that

Reply:
The change is going to be in the CI test rsyslog config?  If so, should I wait for another commit to give lgtm?

Reply:
yes, wait